src/main/java/fr/cnes/sonar/report/providers/IssuesProvider.java

```diff
@@ -218,6 +228,19 @@ private void setIssuesLanguage(Issue[] issues, Rule[] rules) {
                     getServer().getUrl(), getProjectKey(), maxPerPage, page, CONFIRMED, getBranch());
             // perform the request to the server
             final JsonObject jo = request(request);
+
+            // Add the severity "CRITICAL" for the security hotspots
+            // Because the issue type "hotspot security" do not have any severity on the json file
+            JsonArray ja = jo.getAsJsonArray(ISSUES);
+            for(int i = 0; i < ja.size(); i++){
+                JsonElement je = ja.get(i);
+                JsonObject jobj = je.getAsJsonObject();
+                //If there is no severity, it's a security hotspot issue
+                if(!jobj.has(SEVERITY)){
+                    // Add the proper property into the JSON file
+                    jobj.addProperty(SEVERITY, StringManager.HOTSPOT_SEVERITY);
```

Comment:
Does it set the severity ?

Reply:
Yes because I manually add the parameter "severity" in the json response.
Thus, when the code run through this json file, it can find "CRITICAL"
So, it sets the severity to "CRITICAL"